backend/src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.backend
+
+import akka.actor.{ActorRef, Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendActor.{Validate, AfterAll, BeforeAll, CreateJobExecutor}
+import cromwell.backend.model.JobDescriptor
+
+object BackendActor {
+
+  // Commands
+  sealed trait BackendActorMessage
+  case object BeforeAll extends BackendActorMessage
+  case class CreateJobExecutor(jobDescriptor: JobDescriptor) extends BackendActorMessage
+  case object AfterAll extends BackendActorMessage
+  case object Validate extends BackendActorMessage
+
+  // Events
+  sealed trait BackendActorEvent
+  case object BeforeAllCompleted extends BackendActorEvent
+  case class BeforeAllFailed(throwable: Throwable) extends BackendActorEvent
+  case class JobExecutorCreated(backendJobExecutor: ActorRef) extends BackendActorEvent
+  case class JobExecutorCreationFailed(throwable: Throwable) extends BackendActorEvent
```

Comment:
Second one yes but first one you can get that from backendJobExecutor... I think we can define a message in BackendJobExecutorActor for this since in the BackendJobExecutor trait is already defined.